storage/badger/dkg_keys.go

```diff
@@ -66,12 +66,30 @@ func (k *DKGKeys) retrieveTx(epochCounter uint64) func(tx *badger.Txn) (*dkg.DKG
 	}
 }
 
+// InsertMyDKGPrivateInfo insert the DKG private key to database when DKG was completed, and a DKG private key was successfully generated.
 func (k *DKGKeys) InsertMyDKGPrivateInfo(epochCounter uint64, info *dkg.DKGParticipantPriv) error {
 	return operation.RetryOnConflictTx(k.db, transaction.Update, k.storeTx(epochCounter, info))
 }
 
-func (k *DKGKeys) RetrieveMyDKGPrivateInfo(epochCounter uint64) (*dkg.DKGParticipantPriv, error) {
+// InsertNoDKGPrivateInfo insert a record in database to indicate that when DKG was completed, the node failed DKG, there is no DKG key generated.
+func (k *DKGKeys) InsertNoDKGPrivateInfo(epochCounter uint64) error {
```

Comment:
🙏 please document possible error returns during normal operation here as well as in the interface 🙇   https://github.com/onflow/flow-go/blob/df5e576c4203cbcd559adc4b47a5466a8f78bbaa/storage/dkg_keys.go#L11-L12